### PR TITLE
feat: GEOT-156: invite-friends

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -238,3 +238,11 @@ this is how you can use static files in css
     width: 100%;
 }
 ```
+
+# Adding external npm modules
```
Comment (ra8ga) [inline README.md]: pls make sure if its not already in master

Comment (ra8ga) [reply README.md:286]: it's not, it can be merged

Comment (mariusz-kabala) [reply README.md:286]: do you know how to read? if so read fucking readme file in your branch

File: packages/app-andrew/src/components/account/UserName/Form/index.tsx [hunk 2]
```diff
@@ -3,10 +3,8 @@ import styles from './styles.scss'
 import { useForm } from 'react-hook-form'
 import { NFC } from '@gtms/commons/types/nfc.d'
 import { useTranslation } from '@gtms/commons/i18n'
-
-// where I should this one below?
+// @mariusz please review if that (below) is ok, if yes I will remove this comment
```
Comment (rfurmaga) [inline packages/app-andrew/src/components/account/UserName/Form/index.tsx:6]: @mariusz-kabala  place for keeping types for userAccount and name of types  are ok?

File: packages/ui/InviteFriends/index.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import { Button } from '@gtms/ui/Button'
+import useClipboard from 'react-use-clipboard'
+
+export const InviteFriends: FC<{
+  additionalStyles?: string
+}> = ({ additionalStyles }) => {
+  const [isCopied, setCopied] = useClipboard(
+    'http://jedziemyna.pl/openerfestival'
+  )
+  const { t } = useTranslation('inviteFriends')
```
Comment (ra8ga) [inline packages/ui/InviteFriends/index.tsx:14]: @mariusz-kabala  do use use useTranslation this way in UI components?

File: packages/ui/InviteFriends/index.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import styles from './styles.scss'
+import cx from 'classnames'
+import { useTranslation } from '@gtms/commons/i18n'
+import { Button } from '@gtms/ui/Button'
+import useClipboard from 'react-use-clipboard'
+
+export const InviteFriends: FC<{
+  additionalStyles?: string
+}> = ({ additionalStyles }) => {
+  const [isCopied, setCopied] = useClipboard(
```
Comment (ra8ga) [inline packages/ui/InviteFriends/index.tsx:11]: @mariusz-kabala should I write test for that?

File: packages/ui/InviteFriends/index.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        className={cx(styles.wrapper, additionalStyles)}
+        data-testid="invite-friends"
+      >
+        <h2>Zaproś znajomych, bla bla bla, bla bla bla</h2>
+        <Button
+          additionalStyles={styles.btn}
+          onClick={setCopied}
+          data-testid="invite-friends-copy-button"
+        >
+          {!isCopied && t('copyLink')}
```
Comment (ra8ga) [inline packages/ui/InviteFriends/index.tsx:28]: @mariusz-kabala should I write test for that?

**Pull request closed without merging**